x11/xorg-xcb-proto/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem      1.0
 
 name            xorg-xcb-proto
 version         1.13
-revision        1
+revision        2
```

Comment:
Similar comments to above.

Reply:
Some Python scripts (xcbgen) are moved from $prefix/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages to $prefix/opt/local/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages. It might not be a big change for a bump.

Reply:
Gotcha. So one needs the bump, the other doesn't.